decode.go

```diff
@@ -307,12 +307,11 @@ func readData(r *bufio.Reader, order binary.ByteOrder) (kobj *K, err error) {
 			return &K{msgtype, vecattr, timearr}, nil
 		}
 		if msgtype == KT {
-			arr := arr.([]int32)
-			var timearr = make([]Time, veclen)
-			for i := 0; i < int(veclen); i++ {
-				timearr[i] = Time(qEpoch.Add(time.Duration(arr[i]) * time.Millisecond))
```

Comment:
Since time is just the time since the beginning of the day, there is no need to add the epoch. 10:30AM 1970 is equivalent to 10:30AM 2000. Since most systems out there have epoch 1970, I suggest we just return the time since 1970. Thoughts @sv ?